### DISCUSSION TITLE
Feedback Yatga 1

### DISCUSSION THREAD

**Discussion by hvdbm** [TableauBits/Yatga#32] [Ideas]
Déposer ici toutes les erreurs ou problèmes avec la phase 1 de Yatga.

**Comment by jviatteau** [comment]
LE BOUTON POUR RETIRER SES CHANSONS EN MODE CONFORTABLE NE MARCHE PAAAAAAS

**Reply by hvdbm** [reply]
C'est fix (mais pas déployé)

**Comment by hvdbm** [comment]
La connexion automatique à mon compte n'est pas toujours automatique.

**Reply by hvdbm** [reply]
Après réflexion je pense que c'est à cause de la phase de réveil du serveur qui n'est alors pas encore capable de traiter les requêtes. Est-ce qu'il serait possible de le signaler à l'utilisateur ?

**Reply by Ithyx** [reply]
Oui c'est bien ça, on pourrait peut-être envoyer un ping au serveur pour le réveiller, et empêcher l'utilisateur de se connecter tant que le ping n'a pas répondu ?

**Reply by hvdbm** [reply]
On pourrait mettre un pop-up loading ? https://material.angular.io/components/progress-spinner/examples

**Reply by Ithyx** [reply]
à voir ce que ça donnerait

**Reply by hvdbm** [reply]
Ou alors on passe à 1000 dynos sur Heroku

**Comment by jviatteau** [comment]
Ce serait ptêtre cool de mettre un raccourci dans la barre du haut directement vers les constitutions ouvertes auxquelles tu participes

**Reply by jviatteau** [reply]
ah aussi un pop-up au lieu d'un ptit symbole en haut à droite quand tu te fais déco du serveur

**Reply by hvdbm** [reply]
@jviatteau quelque chose comme ça te plairait ? https://material.angular.io/components/snack-bar/examples

**Reply by jviatteau** [reply]
c'est pas mal, en le mettant au milieu avec ptêtre un bouton pour refresh qui vient avec ça serait cool

**Comment by jviatteau** [comment]
ce serait possible d'avoir un truc qui t'indique que le serveur est en train de charger quand tu arrives sur la page mais qu'il tourne pas encore? Parce que ça aiderait les utilisateurs à attendre et pas spammer refresh ou connexion

**Reply by hvdbm** [reply]
Voir https://github.com/TableauBits/Yatga/discussions/32#discussioncomment-2085495
